connectivity/drivers/ble/FEATURE_BLE/COMPONENT_CYW43XXX/CMakeLists.txt

```diff
@@ -1,16 +1,20 @@
 # Copyright (c) 2020 ARM Limited. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
+add_library(mbed-os-ble-cyw43xxx INTERFACE)
+
 mbed_add_cmake_directory_if_labels("TARGET")
```

Comment:
Similar to what was done [elsewhere](connectivity/FEATURE_BLE/libraries/CMakeLists.txt), should you not have :
```c++
if("PS0C6" IN_LIST MBED_TARGET_LABELS)
    add_subdirectory(TARGET_PS0C6)
endif()
```

Reply:
yes, this will be need to be fixed. We do not need util function anymore. I won't have time to fix it before I go for holiday. Would you be able to? I made it for sockets for k64f only to test locally.